commit eece092484557c1a27cea92816f725b877cda3dd
Author: Michal Hrušecký <Michal.Hrusecky@nic.cz>
Date:   Fri Oct 21 14:46:47 2016 +0200

    Log debug output explicitly
    
    Explicitly forward debugging output to the log directory to be inspected later.
    Wouldn't needlessly clutter system log (thought procd) and would make it easier
    to inspect.

diff --git a/legacy/updater.init b/legacy/updater.init
index 4916e01..869c908 100644
--- a/legacy/updater.init
+++ b/legacy/updater.init
@@ -11,7 +11,7 @@ status() {
 
 start() {
 	# Just check if we should resume the updater
-	updater-resume.sh
+	updater-resume.sh 2> /var/log/updater-resume.log
 }
 
 stop() {
